salt/modules/yumpkg.py

```diff
@@ -1853,6 +1856,27 @@ def upgrade(name=None,
 
         .. versionadded:: 2016.3.0
 
+    minimal : False
+        Use upgrade-minimal instead of upgrade (e.g., ``yum upgrade-minimal``)
+        Goes to the 'newest' package match which fixes a problem that affects your system.
+
+        .. code-block:: bash
+
+            salt '*' pkg.upgrade minimal=True
+
+        .. versionadded:: Fluorine
+
+    obsoletes : True
+        Controls wether dnf/yum should take obsoletes into account and remove them.
+        If set to False yum will use update instead of upgrade
+        and dnf will be run with --obsoletes=False
```

Comment:
The above two lines would be a good place to add more emphasis, can you surround "update", "upgrade", and "--obsoletes=False" in double-backticks?